webapp/vite.config.ts

```diff
@@ -1,3 +1,4 @@
+/// <reference types="vitest" />
```

Comment:
Why this line?

Reply:
reference for vitest types, you can read this: https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html#-reference-types-